src/sets.jl

```diff
@@ -58,3 +58,9 @@ julia> @constraint(model, [t, x, x-1, x-2] in RotatedSecondOrderCone())
 """
 struct RotatedSecondOrderCone <: AbstractVectorSet end
 moi_set(::RotatedSecondOrderCone, dim::Int) = MOI.RotatedSecondOrderCone(dim)
+
+# Deprecation for JuMP v0.18 -> JuMP v0.19 transition
+function LinearAlgebra.norm(::AbstractVector{<:AbstractJuMPScalar})
+    error("`@constraint(model, norm(x) <= t)` should now be written as " *
```

Comment:
Clarify that this error message is an example of how to reformulate `norm()`, not that it is a direct suggestion for the user's code. It might be confusing if they already have a variable named `x`. E.g.,
"JuMP no longer performs automatic transformation of norm() expressions into second-order cone constraints. They should now be expressed using the SecondOrderCone() set. For example, ..."